climada/hazard/trop_cyclone.py

```diff
@@ -336,46 +336,93 @@ def from_tracks(
 
     def apply_climate_scenario_knu(
         self,
-        ref_year: int = 2050,
-        rcp_scenario: int = 45
+        percentile: str = '50',
+        rcp_scenario: int = 85,
+        ref_year: int = 2035,
+        baseline: tuple = (1982, 2022)
```

Comment:
Personally and as commented on the previous version, I would not set any default values for all the parameters, in order to force the user to select a scenario, percentile, year and baseline. Also, I am not sure about the name `ref_year`, why not `target_year` instead? Seems more implicit to me.
Finally, I would change the order of the arguments, using in the order: target_year, rcp_scenario, percentile, baseline.